controllers/controllers_test.go

```diff
@@ -118,4 +119,38 @@ var _ = Describe("Identity Controller", func() {
 			Expect(body.SmartMangement.IsEntitled).To(Equal(false), "smart_management.is_entitled expected to be false")
 		})
 	})
+
+	Context("When the account number is -1 or '' ", func() {
+		var fakeResponse SubscriptionsResponse
+
+		BeforeEach(func() {
```

Comment:
Does this need to be done in a BeforeEach? It seems like fakeResponse could be initialized once as a constant, since you want that to be returned every time. I haven't done unit testing in Go yet, so I may be off base here :slightly_smiling_face:

Reply:
@kruai I think you are correct here. I think I was confused on this. I will update the PR to reflect this. :slightly_smiling_face: